en/docs/learn/api-controller/advanced-topics/configuring-different-endpoint-types.md

```diff
@@ -25,7 +25,7 @@ API Manager supports four (4) main types of endpoints as follows.
         - `production`: An array which consists the multiple production endpoints
         - `sandbox`: An array which consists the multiple sandbox endpoints
         - `sessionManagement` (Optional): Values can be `none`, `transport`, `soap`, `simpleClientSession` and if not specified the default value is `transport`
-        - `sessionTimeout` (Optional): The number of milliseconds after which the session would time out
+        - `sessionTimeOut` (Optional): The number of milliseconds after which the session would time out
```

Comment:
We can keep this as sessionTimeout right? Timeout is one word AFAIK.Shall we recheck?

Reply:
APIM needs this as sessionTimeOut. So we should give it as sessionTimeOut. 
Or else we need to change this in code from apictl side to take sessionTimeout and when sending to APIM the .json string should have it as sessionTimeOut.
Even we change like that, if a user mistakenly uses sessionTimeOut in apictl because he/she thinks that the APIM supports that, this will not work.

Reply:
ah okay. Didn't know about this.

Reply:
No worries